builtin/providers/aws/resource_aws_db_instance_test.go

```diff
@@ -49,6 +49,25 @@ func TestAccAWSDBInstance_basic(t *testing.T) {
 	})
 }
 
+func TestAccAWSDBInstance_withoutEngineVersion(t *testing.T) {
```

Comment:
since we aren't specifying it in the configuration, we could simply check that it exists. I would just check that the value returned from the API is not an empty string, and then probably remove this line:

```
resource.TestCheckResourceAttr(
  "aws_db_instance.bar", "engine_version", "5.6.21"),
```

It's computed now, let's just make sure it's there